style.css

```diff
@@ -138,4 +151,523 @@
   background-position: right;
   margin-left: 275px;
   margin-top: -160px;
+  padding-bottom: 240px;
+}
+
+.works-main-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 120px 0;
+  gap: 80px;
+  position: absolute;
+  width: 375px;
+  height: 3455px;
+  left: 0;
+  top: 682px;
+  background-color: #fff;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.recent-works-title {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 24px;
+  width: 327px;
+  height: 107px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 70px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 40px;
+  line-height: 52px;
+  color: #1c1a19;
+  padding-bottom: -60px;
+  border-bottom: 1px solid #212121;
+}
+
+.cards-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 24px;
+  width: 375px;
+  height: 3028px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
```

Comment:
- [ ] Kindly use the CSS technology `display: grid` for the projects and `display: flex` for each project alignment.

Reply:
Fixed.